common/csv_utils.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+)
+
+//Read CSV file
+func ReadCSV(path string) ([][]string, error) {
+	csvFile, err := os.Open(path)
+	if err != nil {
```

Comment:
While it's great to log the errors, if you failed to open the file the rest of the flow will potentially break since you're not ending the flow here. You should probably log the error(as you are doing now), and then return to avoid breaking everything